agent-core/src/main/java/com/newrelic/agent/android/logging/LoggingConfiguration.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2023. New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.newrelic.agent.android.logging;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+import com.google.gson.annotations.SerializedName;
+import com.newrelic.agent.android.AgentConfiguration;
+
+import java.lang.reflect.Type;
+
+public class LoggingConfiguration {
```

Comment:
LoggingConfiguration is the base class for LogReportingConfiguration b/c at the time I thought `agent-level logging control` may be a future request. I'll probably just combine the two classes until that's a requirement.